### PR TITLE
fix(Slider): updated input text styling

### LINKED ISSUE #8380: Bug - Slider: fix styles for value input
**Describe the problem**
Follow up from https://github.com/patternfly/patternfly/issues/5176

### PR DESCRIPTION
**Pull request by thatblindgeye**
**What**: Closes #8380 

Should wait until the Core PR is merged and the patternfly version is bumped to ensure the styles are applied properly for this.

**Additional issues**:

### CODE REVIEW

**Review by mcarrano — APPROVED** [review]
Looks good to me! Thanks @thatblindgeye .

File: packages/react-core/src/components/Slider/Slider.tsx [hunk 1]
```diff
@@ -326,7 +326,10 @@ export const Slider: React.FunctionComponent<SliderProps> = ({
       return (
         <InputGroup>
           {textInput}
-          <InputGroupText variant="plain"> {inputLabel}</InputGroupText>
+          <InputGroupText {...(isDisabled && { className: css(styles.modifiers.disabled) })}>
+            {' '}
```
Comment (mcoker) [inline packages/react-core/src/components/Slider/Slider.tsx:330]: Just curious what this is for? I don't think a space would ever actually render unless there were something before it.

**Review by mcoker — APPROVED** [review]
lgtm!

File: packages/react-core/src/components/Slider/Slider.tsx (hunk 1 above)
Comment (thatblindgeye) [reply packages/react-core/src/components/Slider/Slider.tsx:330]: Ah good catch. It looks like there was a whitespace originally and it got converted to this.

**Review by tlabaj — APPROVED** [review]
LGTM

**Review by gitdallas — APPROVED** [review]

**Pull request merged**